Encodings.md

```diff
@@ -179,6 +179,12 @@ This encoding is adapted from the Binary packing described in
 ["Decoding billions of integers per second through vectorization"](http://arxiv.org/pdf/1209.2137v5.pdf)
 by D. Lemire and L. Boytsov.
 
+Delta encoding is best when used on sorted data, or data with runs of repeated
+values. It can also be useful when the range of values is small, such as would
+be the case with INT_8 data. It should *not* be used when the range of the data
```

Comment:
change should not to should better not? Since though not recommended, user can also do that.